target_stitch/__init__.py

```diff
@@ -613,7 +613,7 @@ def handle_line(self, line):
                 self.contains_activate_version[current_stream] = True
 
             num_bytes = self.buffer_size_bytes
-            num_messages = len(self.messages[current_stream])
+            num_messages = sum((len(messages) for messages in self.messages.values()))
```

Comment:
Does a test cover this case? Just to be extra sure.

Reply:
Just updated the test to check this condition